public/app/pipeline-orchestrator.js

```
@@ -325,6 +328,24 @@ class PipelineOrchestrator {
           );
           // Share with app instance
           if (window._vipApp) window._vipApp.mlWorker = this.mlWorker;
+          // Re-send any isolation config and sound mutes that the user may
+          // have changed before the worker became ready.
+          try {
+            this.mlWorker.postMessage({
+              type: 'setIsolationConfig',
+              payload: this._isolationParams,
+            });
+            const sm = window._vipApp && window._vipApp.soundMutes;
```

ESLint

[error] 'window' is not defined.

(no-undef)